### PR TITLE
feat/adding faq to landing page

### PR DESCRIPTION
**Pull request by kkatusic**
- added all question and answers
- added links to questions

### CODE REVIEW

File: apps/platform/messages/en.json [hunk 1]
```diff
@@ -71,18 +71,30 @@
...
-    "decide": "Who decides which proposals will be accepted for voting and how?",
-    "decideAnswer": "A new voting mechanism is used, called Quadratic Funding. The project with most donations will get the most funding from the City. It allows anyone to vote by donating money to their favourite projects. With every donation, funding is given to project from the matching pool.",
-    "should": "Why should I donate?",
-    "shouldAnswer": "A new voting mechanism is used, called Quadratic Funding. The project with most donations will get the most funding from the City. It allows anyone to vote by donating money to their favourite projects. With every donation, funding is given to project from the matching pool."
+    "zazelenimo": "What is Zazelenimo?",
+    "zazelenimoAnswer1": "Let's go green is an online application for the implementation of participatory budgeting programs in cities.",
+    "zazelenimoAnswer2": "The special feature of Zazelenimo, compared to other applications that have the same purpose, is the use of Ethereum and a special form of voting called square funding.",
```
Comment (b0rza) [inline apps/platform/messages/en.json:76]: 'Zazelenimo' used here, but 'Let's go green' above? Is that ok?

Comment (kkatusic) [reply apps/platform/messages/en.json:76]: @tomislavmamic will you provide English translation and then we can be sure about naming :)

Comment (tomislavmamic) [reply apps/platform/messages/en.json:76]: Let's just use Zazelenimo everywhere. The rest of translation is good to go.

**Review by tomislavmamic — APPROVED** [review]

**Review by kkatusic — COMMENTED** [review]
Thx @tomislavmamic

**Pull request merged**